arangod/Replication/TailingSyncer.cpp

```diff
@@ -470,7 +470,7 @@ Result TailingSyncer::processDocument(TRI_replication_operation_e type,
       // of the result :-|
       TRI_ASSERT(type != REPLICATION_MARKER_REMOVE);
       TRI_ASSERT(!res.errorMessage().empty());
-      conflictDocumentKey = std::move(res.errorMessage());
+      conflictDocumentKey = res.errorMessage();
```

Comment:
```suggestion
      conflictDocumentKey = std::move(res).errorMessage();
```
This allows the errorMessage to be moved out of `res` (see #8157 for details).